library/db/src/main/kotlin/org/cru/godtools/db/room/GodToolsRoomDatabase.kt

```diff
@@ -77,6 +77,7 @@ import org.cru.godtools.db.room.repository.UserRoomRepository
         AutoMigration(from = 19, to = 20),
         AutoMigration(from = 20, to = 21),
         AutoMigration(from = 21, to = 22),
+        AutoMigration(from = 22, to = 23),
```

Comment:
add a db migration test for the migration to ensure that the field is created and initialized to the default value